### PR TITLE
45 landing page

### PR DESCRIPTION
**Pull request by kanenathan213**

### CODE REVIEW

File: javascript/app.js [hunk 1]
```diff
@@ -52,7 +65,7 @@ FinancialFreedom.controller('HeaderController', ['$scope', '$location',  functio
     };
 
     $scope.tabsAreVisible = function() {
-        if ($location.path() == '/about' || $location.path() == '/time-to-retirement') {
+        if ($location.path() == '/about' || $location.path() == '/time-to-retirement' || $location.path() == '/') {
```
Comment (bobo333) [inline javascript/app.js]: if we expect this function to expand to more pages, we could change how this expression is checked by doing something like the following:
`
visible_pages = [
  '/about',
  '/time-to-retirement',
  '/'
]

if ( visible_pages.indexOf($location.path() ) != -1)
`

it's mostly fine as is, this way would just make it a bit shorter and easier to understand and see what pages it's actually checking for

File: sass/partials/_intro.scss [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	background: black;
+	margin: 110px auto auto auto;
+	text-align: center;
+	display: block;
+	background: $dark_green;
+	padding: 20px 30px;
+	@include border-radius(10px);
+	font-size: 1.5em;
+	@include hover-animation(all);
+	cursor: pointer;
```
Comment (bobo333) [inline sass/partials/_intro.scss]: the `@include`s are usually done at the top before other styling just for ease of reading.

Comment (kanenathan213) [inline sass/partials/_intro.scss]: Yep, I can move these up. Same for screen size / responsive @includes?

File: javascript/app.js (hunk 1 above)
Comment (kanenathan213) [inline javascript/app.js]: Yeah good call, I knew it was a short term hack. Check out the way I've done it in the next version.

**Pull request merged**